starter_code/src/movies.js

```diff
@@ -2,14 +2,57 @@
 
 // Iteration 1: Ordering by year - Order by year, ascending (in growing order)
 
+let copy = movies.slice(0, 250);
+let orderByYear = copy.sort(function(a,b) {return a.year - b.year})
+
+
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct
 
+let howManyMovies = movies.filter(function(movie){
+   if(movie.director === "Steven Spielberg" && movie.genre.includes("Drama")){
+       return true;
```

Comment:
here we dont need to write condition with true and false ,instead we can just return movie.director === "Steven Spielberg" && movie.genre.includes("Drama") 
it does the same thing, but as i told you before you think like a programmer and its amazing